tests/certification/go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/cloudwego/kitex v0.5.0
 	github.com/cloudwego/kitex-examples v0.1.1
 	github.com/dapr/components-contrib v1.11.0-rc.1
-	github.com/dapr/dapr v1.10.5-rc.1.0.20230430203526-7a0fdf9f016b
+	github.com/dapr/dapr v1.10.7-0.20230510004949-770d4e51604f
```

Comment:
Should we use 1.11?

Reply:
this PR won't matter anymore when we merge release to master anyway